packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -1587,7 +1587,7 @@ export class StripeHelper {
    */
   async expandResource<T>(
     resource: string | T,
-    resourceType: string
+    resourceType: typeof VALID_RESOURCE_TYPES[number]
```

Comment:
This type tweak isn't strictly necessary after I went through several iterations with this PR. But, it seems a decent improvement? Any module using `expandResource` in Typescript should now get hints as to the proper choices for `resourceType`